packages/bitcore-node/scripts/syncTxnAndCoinsWallets.js

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env node
+'use scrict';
+
+/**
+ * This script fixes an issue where the wallets are in Coin.wallets but not Transaction.wallets in the db
+ */
+
+const fs = require('fs');
+const { Storage } = require('../build/src/services/storage');
+const { TransactionStorage } = require('../build/src/models/transaction');
+const { CoinStorage } = require('../build/src/models/coin');
+
+function usage(err) {
+  if (err) {
+    console.log(err);
+    console.log('');
+  }
+  console.log('Usage: ./fixTxnToWalletMarriage.js [options] <chain> <network> <startBlockHeight> <endBlockHeight>');
```

Comment:
We should change this to the actual name 'fixTxnToWalletMarriage.js' => 'syncTxnAndCoinsWallets.js'